code/modules/projectiles/guns/magnetic/magnetic_railgun_vr.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/weapon/gun/magnetic/railgun/flechette/pistol/khi
+	name = "kitsuhana flechette pistol"
```

Comment:
Personally, I would say Kitsuhana probably should not be going into arms manufacturing? They already have a ton of products on station. Most weapons are manufactured by Hephaestus if you need a different company name, though probably would be fine to just remove the company name entirely.

Reply:
They didn't make it, as the description says, it's just been upgraded. This isn't the normal rail pistol I added either, that is in the other file. This is just a version for admemes.

Reply:
ah, Oki